web/src/components/ui/steps/StepProgress.module.css

```diff
@@ -1,8 +1,15 @@
+.step-progress-container {
+  width: 60%;
+  margin: 0 auto;
+  padding: 1.5rem;
+  border: 1px solid #e5e7eb;
+  border-radius: 0.5rem;
+}
+
 .intro-text {
-  margin: 0 0 1.5rem 0;
-  font-size: 0.875rem;
-  color: #6b7280;
-  line-height: 1.6;
+  margin: 0 0 1.2rem 0;
```

Comment:
might be personal preference but I feel like this + the bottom padding on completion-header makes it look quite far from the line, I'd pick one rather than both